themes/lara/lara-light/teal/theme.scss

```diff
@@ -4,7 +4,7 @@ $primaryDarkColor: #0d9488 !default;
 $primaryDarkerColor: #0f766e !default;
 $primaryTextColor: #ffffff !default;
 
-$highlightBg: #0f766e !default;
+$highlightBg: #f0fffb !default;
```

Comment:
For what it's worth, I think the color should be this, sourced from the primeuix repo and the latest color palette!

#f0fdfa

https://github.com/primefaces/primeuix/blob/4cd80750aa2850690b7942d72d180ec8c4efaa49/packages/themes/src/presets/lara/base/index.ts#L19